include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -80,14 +81,18 @@ class FEEvaluation;
  *
  * @author Katharina Kormann and Martin Kronbichler, 2010-2018
  */
-template <int dim, int n_components_, typename Number, bool is_face = false>
+template <int dim,
+          int n_components_,
+          typename Number,
+          bool is_face,
+          typename VectorizedArrayType>
```

Comment:
I have put everywhere `VectorizedArrayType` to the last position. To be able to force in the `matrixfree` folder that everywhere VectorizedArrayType is passed explicitly to the class `FEEvalutionBase`, I had to remove the `is_face` default value. Since the matrixfree-folder is now working with the additional template parameter, I can revert the default value for external usage. In that case the default parameter would look like this:
```cpp
template <int dim,
          int n_components_,
          typename Number,
          bool is_face = false,
          typename VectorizedArrayType = VectorizedArray<Number>>
class FEEvaluationBase
```

Reply:
Yes, this is what I had in mind!